charts/eap-xp6/values.yaml

```diff
@@ -33,14 +33,14 @@ deploy:
     httpGet:
       path: /health/live
       port: admin
-    initialDelaySeconds: 60
   readinessProbe:
     httpGet:
       path: /health/ready
       port: admin
     initialDelaySeconds: 10
   startupProbe:
     httpGet:
-      path: /health/live
+      path: /health/started
       port: admin
-    initialDelaySeconds: 60
\ No newline at end of file
+    initialDelaySeconds: 10
```

Comment:
while we're at it shouldn't we use WF values?

Reply:
I've aligned the values to have at most 2 minutes for the pod to start.